test/uk/gov/hmrc/vulnerabilities/persistence/RawReportsRepositorySpec.scala

```diff
@@ -48,7 +50,9 @@ class RawReportsRepositorySpec
     "data.transformation-cutoff"    -> "8 days",
   ))
 
-  override lazy val repository = new RawReportsRepository(mongoComponent, configuration)
+  private val appConfig = new AppConfig(new Configuration(ConfigFactory.load()))
```

Comment:
Should the test explicitly define `regex.exclusion` here since it's reliant on specific values?